src/world_renderer.rs

```diff
@@ -88,6 +89,13 @@ impl WorldRenderer {
             self.items.collect(COBBELSTONE);
         }
 
+        // Try to lock the mouse to the window, this doen't work for all OS
+        let lock_mouse = window.set_cursor_grab(CursorGrabMode::Confined)
+            .or_else(|_e| window.set_cursor_grab(CursorGrabMode::Locked));
+        if lock_mouse.is_err() { println!("Can't lock")
```

Comment:
formatting mistake, please let's be careful with this in the future, it's very important on large project.